apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -269,6 +269,11 @@ const EVENTS = {
   PUBLISH_MODEL_CARD_INFO: 'Student publishes their model card info',
 
   // CPA
+  CPA_PARENT_EMAIL_BANNER_CLICKED: 'cpa_parent_email_banner_clicked',
+  CPA_PARENT_EMAIL_BANNER_SUBMITTED: 'cpa_parent_email_banner_submitted',
+  CPA_PARENT_EMAIL_BANNER_UPDATED: 'cpa_parent_email_banner_updated',
+  CPA_PARENT_EMAIL_BANNER_RESEND: 'cpa_parent_email_banner_resend',
+  CPA_PARENT_EMAIL_BANNER_CLOSED: 'cpa_parent_email_banner_closed',
```

Comment:
Although it wasn't originally requested, I think we will also want 
```
CPA_PARENT_EMAIL_BANNER_SHOWN: 'cpa_parent_email_banner_shown',
```
This is the first step in the experience. Please add this metric. I updated the JIRA task.

Reply:
https://github.com/code-dot-org/code-dot-org/pull/58434/commits/0e74324359b554b5b92108e76e8065710bd0d3e0